test/test_ugrid.py

```diff
@@ -93,9 +99,9 @@ def test_standardized_dtype_and_fill_dask(self):
 
         with dask chunking
         """
-        ug_filename = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30.ug"
+        ug_filename = current_path / "meshfiles" / "ugrid" / "outRLL1deg" / "outRLL1deg.ug"
         xr_grid = xr.open_dataset(str(ug_filename), chunks={'nMesh2_node': 100})
```

Comment:
What will be generated if the input `ug_filename` doesn't have any value (an empty file)?

Reply:
It should raise an exception according to the Xarray API

Reply:
Then probably we can use a `try catch` here

Reply:
I'm not fully understanding what the concern here is though, at the load dataset line?

Reply:
What I meant is inside our `standardized_dtype_and_fill_dask():`, did we handle the case when the input dataset is empty?

Reply:
I think this is beyond the scope of this PR. If we load in an empty, or incomplete dataset, it should be caught by `parse_grid` within `grid.py`

Reply:
So if we can make sure `parse_grid` will prevent empty or incomplete dataset, then we're fine here.